lib/generators/rspec/scaffold/templates/request_spec.rb

```diff
@@ -16,6 +16,11 @@
 RSpec.describe "/<%= name.underscore.pluralize %>", <%= type_metatag(:request) %> do
   # <%= class_name %>. As you add validations to <%= class_name %>, be sure to
   # adjust the attributes here as well.
+  <% if mountable_engine? -%>
+    include Engine.routes.url_helpers
```

Comment:
I think this needs to be fully qualified? Not quite sure how you'd get that, maybe we can comment it out with a note if we can't (so the spec will run, but need editing)

Reply:
I think we want something like `<%= namespace %>::Engine`

Reply:
@JonRowe does it need to be fully qualified since it's already namespaced in the module on L15? I'm not opposed to adding it, just curious.

Reply:
Hm I guess not but its more clear.